tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "declaration": true,
     "strict": true
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts", "src/*.ts"],
+  "exclude": ["node_modules"],
+  "sourceRoot": "./src",
+  "sourceMap": true
```

Comment:
Do we want source maps in production? IMO we already have this in `tsconfig.development.json` to prevent emitting source maps to production.

Reply:
Also I thing this is a `compilerOptions`, so it's on the wrong place here

Reply:
Got it thanks